docs/kickstart-docs.rst

```diff
@@ -1047,6 +1047,19 @@ Create a logical volume for Logical Volume Management (LVM).
     ``--size=``. This option must be used with
     ``--useexisting --size=``, or an error will be raised.
 
+``--cachesize``
+
+    Requested size of cache attached to the logical volume. (Requires --cachepvs)
```

Comment:
Good catch, thanks! It's megabytes as all our storage sizes, but I'm definitely going to make it explicit in the docs.